src/Sushi.Mediakiwi.API/Controllers/BaseMediakiwiApiController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Sushi.Mediakiwi.API.Filters;
+using System;
+
+namespace Sushi.Mediakiwi.API.Controllers
+{
+    [TypeFilter(typeof(MediakiwiConsoleFilter))]
+    public abstract class BaseMediakiwiApiController : ControllerBase
+    {
+        protected Data.IApplicationUser MediakiwiUser
+        {
+            get
+            {
+                if (HttpContext.Items.ContainsKey(Common.API_USER_CONTEXT))
+                {
+                    return HttpContext.Items[Common.API_USER_CONTEXT] as Data.IApplicationUser;
+                }
+                else
+                {
+                    return default;
```

Comment:
don't use default, it hides intent. I assume this returns NULL?

Reply:
Agreed, adjusted